assets/src/edit-story/components/popup/test/popup.js

```diff
@@ -44,7 +44,6 @@ describe('Popup', () => {
 
     const popup = getByText('Hello World!');
     expect(popup).toBeDefined();
-    expect(popup).toHaveAttribute('width', '270');
```

Comment:
Could we perhaps add a new test with some params for it, now when it supports different positioning options.